libs/browser/src/keybindings.rs

```diff
@@ -0,0 +1,45 @@
+use gtk::gdk;
+use gtk::prelude::*;
+
+use crate::device::Device;
+use crate::options::Options;
+
+/// Set up handler for key press events, e.g. Ctrl+R to reload the page.
+pub fn attach(options: Options, device: &Device) {
+    device.window.connect_key_press_event({
+        let device = device.clone();
+
+        move |_, event| {
+            let modifiers = event.state();
+            match event.keyval() {
+                gdk::keys::constants::w if modifiers.contains(gdk::ModifierType::CONTROL_MASK) => {
+                    gtk::main_quit();
+                    gtk::glib::Propagation::Stop
+                }
+                gdk::keys::constants::r if modifiers.contains(gdk::ModifierType::CONTROL_MASK) => {
+                    device.reload();
+                    gtk::glib::Propagation::Stop
+                }
+                gdk::keys::constants::Escape => {
+                    if options.is_dev() {
+                        if let Err(e) = device.unfullscreen() {
+                            eprintln!("Error unfullscreening: {e}");
+                        }
```

Comment:
Maybe crash as well as log since it's hard to see this log if you're in fullscreen